src/main/groovy/org/gradlefx/tasks/AirPackage.groovy

```diff
@@ -65,6 +69,18 @@ class AirPackage extends DefaultTask {
         showAntOutput ant.properties[ANT_OUTPUT_PROPERTY]
     }
 
+    def initInputOutputFiles() {
+        def swfFileName
+        if (flexConvention.air.mainSwfDir) {
+            swfFileName = new File(project.buildDir.path, "${flexConvention.air.mainSwfDir}/${flexConvention.output}.${FlexType.swf}").toString()
+        } else {
+            swfFileName = "${project.buildDir.path}/${flexConvention.output}.${FlexType.swf}"
+        }
+
+        inputs.files swfFileName
+        outputs.files project.file(project.buildDir.name + '/' + flexConvention.output).absolutePath
```

Comment:
I realized I mixed some conventions in constructing the filenames, so it could be made more consistent.